ml_engine/online_prediction/predict_test.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2016 Google Inc. All Rights Reserved. Licensed under the Apache
```

Comment:
2017, also, these headers still seem different from the ones in the rest of the repo.